test/metarpheus.test.ts

```diff
@@ -1,32 +1,37 @@
 import * as path from 'path';
-import { runCommands, templateDir } from './utils';
+import { runCommands, testAppDir } from './utils';
 import * as fs from 'fs';
 import * as rimraf from 'rimraf';
 
-const config = require('./template-app/metarpheus.config.test');
+const config = require(path.resolve(testAppDir, 'metarpheus.config.test'));
 
 const metarpheusDir = config.apiOut
   .split('/')
   .slice(0, -1)
   .join('/');
-const metarpheusOutPath = path.resolve(templateDir, metarpheusDir);
+const metarpheusOutPath = path.resolve(testAppDir, metarpheusDir);
 
 afterAll(() => {
   rimraf.sync(metarpheusOutPath);
 });
 
-describe('metarpheus', () => {
+fdescribe('metarpheus', () => {
```

Comment:
what’s fdescribe?

Reply:
good catch, it's a leftover (it should `f`ocus on the single suite of tests, but I think it only has effects inside the single file so doesn't do nothing here)